scripts/globals/npc_util.lua

```diff
@@ -284,6 +284,41 @@ function npcUtil.giveItem(player, items, params)
     return true
 end
 
+--[[
+    For handling of 1/1 usable items delivered in a "just used" state. Most 1/1 charged items are Rare,
+    but some exceptions do exist. Pass a value of 1 in notRare for those exceptions.
+    e.g. npc_utils.giveUsedItem(player, 22132, 1) will grant the player an Ex Artemis Bow +1.
+--]]
+function npcUtil.giveUsedItem(player, item, notRare)
```

Comment:
👍

Reply:
looks like that param should just be true/false rather than a number?

Reply:
Functionally the same, but will swap this to a bool instead.